tests/publisher/tests_register_name.py

```diff
@@ -2,12 +2,6 @@
 from tests.publisher.endpoint_testing import BaseTestCases
 
 
-class GetRegisterNamePageNotAuth(BaseTestCases.EndpointLoggedOut):
```

Comment:
Should this be removed? We still have `/register-snap` page, so seems valid to check if user is required to be logged in when accessing it?